crates/sui-benchmark/src/benchmark_setup.rs

```diff
@@ -262,21 +254,24 @@ impl Env {
                     &keystore_path
                 ))
             })?;
+
         let keypair = Arc::new(get_ed25519_keypair_from_keystore(
             keystore_path,
-            &primary_gas_account,
+            &current_gas_account,
         )?);
 
-        let primary_coin = (
-            primary_gas_obj.compute_object_reference(),
-            primary_gas_account,
+        let primary_gas = (
```

Comment:
Why call this primary_gas instead of current_gas? I thought primary_... means what are available at genesis, and current_... means what will be used by the workload. But I may be misunderstanding.

Reply:
Yup, thats right. Changed naming from primary -> current